closure/compiler/closure_js_binary.bzl

```diff
@@ -67,12 +68,20 @@ def _impl(ctx):
     args += ["--debug"]
   elif is_using_closure_library(srcs):
     args += ["--define=goog.DEBUG=false"]
+  if is_using_closure_library(srcs) and language_out in ["ECMASCRIPT6_TYPED",
+                                                         "ECMASCRIPT6_STRICT",
+                                                         "ECMASCRIPT5_STRICT"]:
+    args += ["--define=goog.STRICT_MODE_COMPATIBLE"]
+  if contains_file(srcs, ctx.attr._soyutils_usegoog):
```

Comment:
Ok that makes sense. I guess it wouldn't be so terrible to have such a tiny dependency. But in that case, you probably want to change `_soyutils_usegoog` to have `single_file=True` and then this parameter should be `ctx.file._soyutils_usegoog.short_path` to avoid the type error.

I would recommend writing a failure test to verify your change is correct, but I haven't gotten around to adding an `internal_expect_failure` flag to `closure_template_js_library` yet, which is currently only available on `closure_js_library`.